app/config/config.yml

```diff
@@ -8,9 +8,6 @@ parameters:
     standardEmptyOnNoArticles: "/common/lists/listScrollShopArticle.html.twig"
     standard:                  "/common/lists/listScrollShopArticle.html.twig"
 
-chameleon_system_cms_text_field:
```

Comment:
The default is (still?) true in \ChameleonSystem\CmsTextFieldBundle\DependencyInjection\Configuration.